sdk/highlight-py/e2e/highlight_flask/app.py

```diff
@@ -1,29 +1,37 @@
 import logging
 import random
 import time
+import requests
 
 from flask import Flask, request
 
 import highlight_io
 from highlight_io.integrations.flask import FlaskIntegration
+from highlight_io.integrations.requests import RequestsIntegration
 
 app = Flask(__name__)
 H = highlight_io.H(
     "1",
-    integrations=[FlaskIntegration()],
+    integrations=[FlaskIntegration(), RequestsIntegration()],
     instrument_logging=True,
     otlp_endpoint="http://localhost:4318",
     service_name="my-flask-app",
     service_version="1.0.0",
     environment="e2e-test",
+    tracing_origins=["localhost", "highlight.io"],
```

Comment:
does this option fit better in the `RequestsIntegration()` itself? that makes more sense to me since that's what is actually able to trace a request (outgoing http request), and if there is a future other external service integration (like `GRPCIntegration()`) that may have a different set of tracing origins

i propose moving the option into the ingration itself like so: `RequestsIntegration(tracing_origins=[...])`

Reply:
That makes sense to me. I was originally thinking that these tracing origins would be across multiple integrations to avoid duplicative code if you were using multiple libraries to make requests, but could see why you may want a different list for grpc vs http.

r.e. out conversation of having these library auto-instrumentations on by default - my thought is:
1. If not included in `integrations` list -> RequestsInstrumentation is `on` with no tracing_origins
2. If added to exclusions list -> RequestsInstrumentation is `off`
3. If added to `integrations` list with tracing origins -> use those integration parameters and leave out the default integration